src/Sylius/Behat/Page/Shop/PaymentRequest/PaymentRequestNotifyPageInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Page\Shop\PaymentRequest;
+
+use FriendsOfBehat\PageObjectExtension\Page\PageInterface as BasePageInterface;
+use Symfony\Component\BrowserKit\AbstractBrowser;
+
+interface PaymentRequestNotifyPageInterface extends BasePageInterface
+{
+    /**
+     * Calls a URI.
+     *
+     * @param string      $method        The request method
+     * @param array       $urlParameters The url parameters
+     * @param array       $files         The files
+     * @param array       $server        The server parameters (HTTP headers are referenced with an HTTP_ prefix as PHP does)
+     * @param string|null $content       The raw body data
+     */
+    public function openWithClient(
+        string $method,
+        array $urlParameters = [],
+        array $files = [],
+        array $server = [],
+        ?string $content = null,
+    ): void;
```

Comment:
```suggestion
    public function openWithClient(
        string $method,
        array $urlParameters = [],
    ): void;
```
From what I can see only these 2 are used atm

Reply:
@NoResponseMate 
I agree but I think about plugins using this. I already struggle in the past to make this working using Sylius Behat services. For example I will need to set server headers to the request to make Stripe works with this Behat Page service.

Reply:
Fair enough 👋
The docblock can go away though, the parameters' names are pretty much self-explanatory